.classpath

```diff
@@ -11,7 +11,5 @@
 			<attribute name="javadoc_location" value="jar:platform:/resource/2017-2706-Robot-Code/lib/non-robot-libs/google/docs.zip!/"/>
 		</attributes>
 	</classpathentry>
-	<classpathentry kind="var" path="USERLIBS_DIR/CTRLib.jar"/>
```

Comment:
Eclipse did it automatically. I don't seem to have the proper modules for them installed. How can I get them installed?